Engine/Fuego/Application.cpp

```diff
@@ -28,31 +28,24 @@ Application::Application() : d(new ApplicationImpl())
     d->m_Running = true;
 
     // Temp::
-    float vertices[3 * 3] = {
-        -0.5f, -0.5f, 0.0f, 0.5f, -0.5f, 0.0f, 0.0f, 0.5f, 0.0f};
+    float vertices[3 * 3] = {-0.5f, -0.5f, 0.0f, 0.5f, -0.5f, 0.0f, 0.0f, 0.5f, 0.0f};
```

Comment:
For Objects like this we have to turn off clang-format -> format how we like it and then turn it on
{ 1, 1, 1,
  0, 0, 0,
}
I dont think so we can automate it but we can agree on style for such cases